File: engine/engine-model/src/main/scala/net/server/channel/packet/command/admin/PlayerWarnPacket.scala
package net.server.channel.packet.command.admin

import net.server.MaplePacket

class PlayerWarnPacket( private var _mode: Byte,  private var _victim: String,  private var _message: String) extends BaseAdminCommandPacket(_mode = _mode) {
     def victim: String = _victim
     def message: String = _message
}

File: engine/engine-model/src/main/java/net/server/channel/packet/command/admin/PlayerWarnPacket.java
package net.server.channel.packet.command.admin;

public class PlayerWarnPacket extends BaseAdminCommandPacket {
   private final String victim;

   private final String message;

   public PlayerWarnPacket(Byte mode, String victim, String message) {
      super(mode);
      this.victim = victim;
      this.message = message;
   }

   public String victim() {
      return victim;
   }

   public String message() {
      return message;
   }
}
